### DISCUSSION TITLE
steam: integrated web browser breaks with --trace or --tracelog

### DISCUSSION THREAD

**Discussion by IcyAlmond** [netblue30/firejail#5319] [Q&A] [answered]
I was trying to figure out a problem with firejail+steam (games with EAC don't load when steam is run with firejail)

But if i run firejail with --trace OR --tracelog, steam's tab content will all become black. I thought maybe i have to specify the profile but I added `--profile=/etc/firejail/steam.profile` but the tab views were all still just black.

**Comment by ghost** [comment]
There's a note in [steam.profile](https://github.com/netblue30/firejail/blob/master/etc/profile-m-z/steam.profile#L160) that accompanies not using tracelog. Can't tell exactly why that is the case, but you might be out of luck here...

**Comment by rusty-snake** [comment, marked as answer]
Firefox and Chromium are known to break with trace(log). If steam uses Gecko/Blink for it's content tab ...

trace(log) is implemented with an preloaded library that wraps certain library calls (`open`, `access`, `stat`, ...). Some programs do not like messing with symbols. There's nothing you can do.
